Dockerfile.deploy

```diff
@@ -52,6 +53,11 @@ RUN apt-get update && apt-get install -y \
         pngcrush                         \
     && rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update && apt-get -t stretch-backports install -y \
+        # For git-based files storage backend
+        libgit2-dev \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Maybe we can merge this "install" into the install command above? That saves us one round of "apt-get update". But since I'm late, I'm OK if this is done next time when we need to update this file. :)

Reply:
I put this into a separate line to only have one `install` target `stretch-backports`, the rest uses the regular packages. I kinda thought that made sense from a stability standpoint.